rasa/core/run.py

```diff
@@ -207,10 +207,13 @@ async def clear_model_files(_app: Sanic, _loop: Text) -> None:
         if app.agent.model_directory:
             shutil.rmtree(_app.agent.model_directory)
 
+    async def run_telemetry(_app: Sanic, _loop: AbstractEventLoop) -> None:
+        telemetry.track_server_start(input_channels, endpoints, app.agent)
+
     app.register_listener(clear_model_files, "after_server_stop")
+    app.register_listener(run_telemetry, "after_server_start")
```

Comment:
This will get fired `number_of_sanic_workers` times 😬 (btw: the number of sanic workers would be interesting to track as well 🕵️ )

Reply:
mhm that is interesting 🤔 I'll move this to a different location then

Reply:
we use `run_operation_in_single_sanic_worker` in Rasa X if that is of any help